backend/src/Edna/Dashboard/Web/API.hs

```diff
@@ -71,6 +94,23 @@ data DashboardEndpoints route = DashboardEndpoints
       :> QueryParam "sortby" StubSortBy
       :> Get '[JSON] ExperimentsResp
 
+  , -- | Get experiment's metadata by ID
+    deGetExperimentMetadata :: route
+      :- "experiment"
+      :> Summary "Get experiment's metadata by ID"
+      :> Capture "experimentId" ExperimentId
+      :> "metadata"
+      :> Get '[JSON] ExperimentMetadata
+
+  , -- | Download experiment data file that stores experiment with given ID
+    deGetExperimentFile :: route
+      :- "experiment"
+      :> Summary "Download experiment data file that stores experiment with given ID"
+      :> Capture "experimentId" ExperimentId
+      :> "file"
+      :> Get '[OctetStream]
+        (Headers '[Header "Content-Disposition" Text] ExperimentFileBlob)
```

Comment:
Is it possible somewhere to get file name and description? But as I see there is no such information in design

Reply:
Yes, if you expand the table in Dashboard
![image](https://user-images.githubusercontent.com/2556409/114177111-b0533100-9944-11eb-82d5-1c6e73a81f67.png)

`Metadata` shows description and metadata from `.xlsx`, `Download` actually downloads `.xlsx` file.

Reply:
And file name?

Reply:
I put file name into `Content-Disposition` header, so that when you click "Download" this file name will be used by default in the file save dialog.